bot.py

```diff
@@ -2046,13 +2046,16 @@ def create_quote_image(id, type, quote, name):
 
     #Is someone being profane?
     if message.channel.name != "serious_business":
-        content = message.content.lower().replace(' ', '')
+        content = message.content.lower()
         for profane_word in profane_words:
             if profane_word in content:
-                if random.randint(1, 50) == 1:
-                    msg = "Hey {member}! Shame on you. This is a christian server.\nSister <@194703127868473344> is always watching.".format(member=member.mention)
+                if random.randint(1, 1) == 1:
```

Comment:
lol why? this means it'll always run

Reply:
Oversight. I changed it for testing and forgot to revert.